ncurses.yaml

```diff
@@ -158,9 +158,7 @@ update:
   enabled: true
   version-transform:
     - match: _p
-      replace: .
-    - match: \-
-      replace: .
+      replace: \-
```

Comment:
I think the match and replace are the wrong way around here: we want to convert a `-` in the upstream version to `_p` in our version.

Reply:
Sorry, I'm the person who introduced this extra code.  I failed to take into consideration the fact that we deliberately use `_p` in our version (I don't really understand why, though...), so you're correct that we need to convert `-` to `_p` for the comparison to work.

Reply:
> I don't really understand why [we embed _p in our version]

Likely to match the versioning used in alpine: https://pkgs.alpinelinux.org/package/edge/main/x86_64/ncurses